public/locales/de-DE/grafana.json

```diff
@@ -1031,6 +1031,9 @@
     "incidents": {
       "title": "Störungen"
     },
+    "incident": {
```

Comment:
Please revert all changes to non-english language files. These translations exists within Crowdin (we can change that), and would just be overridden on next sync.

Reply:
done, thanks